stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/cookbook/ui/AttachmentGallery.kt

```diff
@@ -34,8 +33,5 @@ class AttachmentGallery : Fragment() {
 
         // Set the data to display
         destination.setData(attachmentGalleryItems = listOf(), attachmentIndex = 0)
-
-        // Fire the navigation request
-        ChatUI.instance().navigator.navigate(destination)
```

Comment:
Why it's deleted? How to navigate to destination?

Reply:
Fixed